Gopkg.toml

```diff
@@ -55,6 +57,14 @@
   branch = "release-4.3"
   source = "https://github.com/openshift/cluster-api-provider-azure"
 
+[[constraint]]
+  name = "github.com/jim-minter/go-cosmosdb"
+  revision = "323658c998227f194221880018cfd7a199fd714c"
```

Comment:
`branch = "master"` ?

Reply:
it doesn't seem to require it, but i thought you wanted to fix these generators? so ci didn't misbehave?

Reply:
so you can't specify master and revision "dep ensure" barfs
i thought the point was to nail down the version so we would have to update it?

Reply:
I am being a dummy, change coming..